tests/data/generator/t_protocol/protocol.yaml

```diff
@@ -5,12 +5,13 @@ description: A protocol for testing purposes.
 license: Apache-2.0
 aea_version: '>=0.5.0, <0.6.0'
 fingerprint:
```

Comment:
where is the specification for this protocol?

Reply:
`tests/data/sample_specification,yaml`